projects/unix/Makefile

```diff
@@ -424,6 +432,7 @@ targets:
 	@echo "    NO_ASM=1      == build without inline assembly code (x86 MMX/SSE)"
 	@echo "    USE_GLES=1    == build against GLESv2 instead of OpenGL"
 	@echo "    NO_SSE=1      == build without SSE support"
+	@echo "    USE_FRAMESKIPPER=1 == build with frameskipper feature"
 	@echo "    APIDIR=path   == path to find Mupen64Plus Core headers"
```

Comment:
I tend to prefer one commit per feature as most diff tools allow you to skip space change to focus on code change (I know I know, my combiner refactoring does not follow this statement...). But I don't see any _real_ problem to do cosmetic changes in separate commits. :)